src/fastoad/io/configuration/tests/data/conf_sellar_example/functions.py

```diff
@@ -41,6 +41,11 @@ def setup(self):
 class FunctionF(om.ExplicitComponent):
     """An OpenMDAO component to encapsulate Functions discipline"""
 
+    def initialize(self):
+        # These options have no effect and are used for checks
+        self.options.declare("dummy_f_option", types=int, default=0)
+        self.options.declare("dummy_generic_option", types=str, default="")
+
```

Comment:
That may be me being too picky, but if the generic option resembles something like the propulsion_id, should in also be in `FunctionFAlt`

Reply:
Granted :) (62e98f001f07690301be7b7e1bc097ce85b636d2)